homeassistant/components/asuswrt/__init__.py

```diff
@@ -62,6 +66,23 @@
 async def async_setup(hass, config):
     """Set up the asuswrt component."""
 
+    await async_setup_retry(hass, config)
```

Comment:
Please follow the tibber example instead.

https://github.com/home-assistant/core/blob/58ae117bb415e949a65d61abe3b1a19bca9ebca0/homeassistant/components/tibber/__init__.py#L28

Reply:
Implemented as requested. Still have the problem that checks fail (issue #33524)